database/datastore.go

```diff
@@ -80,10 +82,12 @@ func (d *DatastoreDatabase) GetJobs(parentID string) ([]providers.Job, error) {
 	ctx := context.Background()
 	query := datastore.NewQuery(d.kind).Namespace(d.namespace).Filter("ParentID =", parentID)
 	_, err := d.client.GetAll(ctx, query, &jobs)
+	if len(jobs) == 0 {
```

Comment:
no, `err` is nil when nothing is found. but I'll change the order because len is 0 when `err != nil` too